### ISSUE TITLE
Save path not correct

### ISSUE THREAD

**Issue by jharaldson** [OpenEndedGroup/Field2#85]
Open a field worksheet:
./f fieldbox.FieldBox -file path/something.field2 -threaded 1

When saving the file the path will be:
field_workspace_path/path/something.field2

Correct behavior should be to save the sheet to the same path as the sheet was opened from

**Comment by marcdownie** [comment]
Does an absolute path work as expected?

`./f fieldbox.FieldBox -file /path/something.field2 -threaded 1`

**Comment by jharaldson** [comment]
No. It opens the sheet as expected, but doesn't save it in the right path.

**Comment by marcdownie** [comment]
All right then! Bug it is.
